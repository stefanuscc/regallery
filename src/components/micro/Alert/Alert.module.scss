.alert{
    display: flex;
    justify-content: center;
    align-items: center;
    margin-bottom: 1rem;
    padding: .8rem 1rem;

    background-color: rgb(254 242 242);

    .icon {
        flex-shrink: 0;
        width: 1.25rem;
        height: 1.25rem;
        color: rgb(248 113 113);
    }

    .message {
        margin: 0;
        margin-left: .75rem;
        font-size: .875rem;
        font-weight: 500;
        color: rgb(153 27 27);
    }
}