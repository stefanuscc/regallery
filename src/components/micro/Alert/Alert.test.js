import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Alert from './Alert';
import "../../../setupTests"


test('Alert to match snapshot', () => {
    const component = renderer.create(
        <Alert message="There was some errors on the endpoint, please be patient."/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('Alert message is persistence', () => {
    const alert = shallow(<Alert message="There was some errors on the endpoint, please be patient."/>);

    expect(alert.find('h3').text()).toEqual('There was some errors on the endpoint, please be patient.');
});