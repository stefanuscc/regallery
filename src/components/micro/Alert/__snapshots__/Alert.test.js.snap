// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert to match snapshot 1`] = `
<div
  className="alert undefined"
>
  <svg
    className="icon"
    fill="currentColor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clipRule="evenodd"
      d="M10 18a8 8 0 100-16 8 8 0 000 16zM8.707 7.293a1 1 0 00-1.414 1.414L8.586 10l-1.293 1.293a1 1 0 101.414 1.414L10 11.414l1.293 1.293a1 1 0 001.414-1.414L11.414 10l1.293-1.293a1 1 0 00-1.414-1.414L10 8.586 8.707 7.293z"
      fillRule="evenodd"
    />
  </svg>
  <h3
    className="message"
  >
    There was some errors on the endpoint, please be patient.
  </h3>
</div>
`;
